src/r_things.c

```diff
@@ -669,12 +773,14 @@ static void R_ProjectSprite (mobj_t* thing)
     vis->startfrac += vis->xiscale*(vis->x1-x1);
   vis->patch = lump;
 
+  vis->brightmap = R_GetBrightmap(thing->sprite, frame, rotation);
+
   // get light level
   if (thing->flags & MF_SHADOW)
     vis->colormap[0] = vis->colormap[1] = NULL;               // shadow draw
   else if (fixedcolormap)
     vis->colormap[0] = vis->colormap[1] = fixedcolormap;      // fixed map
-  else if (thing->frame & FF_FULLBRIGHT)
+  else if (thing->frame & FF_FULLBRIGHT && !brightmaps) // TODO
```

Comment:
I've disabled `FF_FULLBRIGHT` for now, it looks like we need some kind of config file for brightmaps, like this: https://github.com/NightFright2k19/gzdoom_brightmaps/blob/master/bmplus_vanilla/brightmaps/sprites/doom2.bm

Reply:
`FF_FULLBRIGHT` is just a flag that we can remove or set at will, depending on the presence of a brightmap for the sprite. However, we should keep a copy if we want to allow in-game toggle. 🤔